web/core/components/issues/issue-detail/issue-activity/activity/activity-list.tsx

```
@@ -42,6 +43,11 @@ export const IssueActivityItem: FC<TIssueActivityItem> = observer((props) => {
   const componentDefaultProps = { activityId, ends };
 
   const activityField = getActivityById(activityId)?.field;
+
+  if (activityField && activityField.startsWith("Custom Property")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)